openmp/runtime/tools/summarizeStats.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
This is not necessary. LLVM requires python 3.8 minimum, and this is not a script I'd expect for us to give de facto support to older versions, and certainly not python 2.